apps/cli/src/live-common-setup-base.ts

```diff
@@ -105,6 +105,7 @@ setSupportedCurrencies([
   "zksync_sepolia",
   "mantra",
   "xion",
+  "sui",
```

Comment:
Should we add `sui_testnet` here as well?

Reply:
Good question actually i didn't figured out why some coins needs this _testnet version. Possible because some coins haas different configs for mainnet and testnet tokens. I didn't spot any difference for now for sui so it looks like we don't need it and is better to remove sui_testnet from everewhere?

Reply:
I don't know, to be honest. Let's add testnet everywhere for consistency.